tests/test_api.py

```diff
@@ -152,19 +116,30 @@ async def test_api_resource_updated(client):
             "last_modified": datetime.now().isoformat(),
         },
     }
-    resp = await client.post("/api/resource/updated/", json=payload)
+    client_http_method: Callable = getattr(
+        client, route["method"]
+    )  # TODO: can be removed once we don't use legacy route anymore
```

Comment:
Nit: redundant `# TODO` as you already have it on line 99 before the test itself?

Reply:
I think better to keep it as I won't be able to remmeber what was `client_http_method` for and that it will be useless later :) At least for me!